panel/package.json

```diff
@@ -20,13 +20,13 @@
     "autosize": "^5.0.1",
     "dayjs": "^1.10.7",
     "mitt": "^3.0.0",
-    "prosemirror-commands": "^1.1.10",
+    "prosemirror-commands": "^1.2.1",
     "prosemirror-history": "^1.2.0",
     "prosemirror-inputrules": "^1.1.3",
-    "prosemirror-keymap": "^1.1.4",
-    "prosemirror-model": "^1.14.3",
+    "prosemirror-keymap": "^1.1.5",
+    "prosemirror-model": "^1.16.1",
     "prosemirror-schema-list": "^1.1.6",
-    "prosemirror-view": "^1.20.1",
+    "prosemirror-view": "^1.23.6",
     "vue": "^2.6.14",
     "vuedraggable": "^2.24.3",
     "vuelidate": "^0.7.6",
```

Comment:
0.7.7?

Reply:
@afbora can you find that there https://github.com/vuelidate/vuelidate/releases? I can't even find 0.7.6 - only 0.7.5 😆

Reply:
Yes, I'm not sure why not listed but you can see on npm:

https://www.npmjs.com/package/vuelidate